src/components/Graph.react.js

```diff
@@ -494,13 +505,7 @@ PlotlyGraph.propTypes = {
 };
 
 PlotlyGraph.defaultProps = {
-    /* eslint-disable no-magic-numbers */
-    id:
-        'graph-' +
-        Math.random()
-            .toString(36)
-            .substring(2, 7),
-    /* eslint-enable no-magic-numbers */
+    id: generateId(),
```

Comment:
What about multiple graphs in the same page? They way they will all have the same id. Similar to when this was added to the dash-table.